pkg/controller/installation/prometheusRules.go

```diff
@@ -35,25 +35,15 @@ func (r *ReconcileInstallation) newAlertsReconciler(logger *logrus.Entry, instal
 				GroupName: fmt.Sprintf("%s-installation.rules", installationName),
 				Rules: []monitoringv1.Rule{
 					{
-						Alert: fmt.Sprintf("%sInstallationControllerIsNotReconciling", strings.ToUpper(installationName)),
+						Alert: fmt.Sprintf("%sInstallationControllerIsInReconcilingErrorState", strings.ToUpper(installationName)),
 						Annotations: map[string]string{
 							"sop_url": resources.SopUrlAlertsAndTroubleshooting,
-							"message": fmt.Sprintf("%s operator has not reconciled successfully in the interval of 15m over the past 1 hour", strings.ToUpper(installationName)),
+							"message": fmt.Sprintf("%s operator has finished installing, but has been in a error state while reconciling in the interval of 5m over the past 1 hour", strings.ToUpper(installationName)),
 						},
-						Expr:   intstr.FromString(fmt.Sprintf("%s_status{stage='complete'} AND on(namespace) rate(controller_runtime_reconcile_total{controller='installation-controller', result='success'}[15m]) == 0", installationName)),
+						Expr:   intstr.FromString(fmt.Sprintf("%s_status{stage='complete'} AND on(namespace) rate(controller_runtime_reconcile_total{controller='installation-controller', result='error'}[5m]) >= 1", installationName)),
```

Comment:
Hey @austincunningham @pepedocs I was thinking about this query a little more. The check is whether in the last hour the expression is satisfied over 5 minutes. If it happened in the last hour then it could be the case that it's resolved itself and the alert notification wouldn't be of much value at that stage. 
The alert could be simplified to `rhoam_status{stage='complete'} AND on(namespace) controller_runtime_reconcile_total{controller='installation-controller', result='error'} >=1` and the for entry changed to `For: 5m`, that way the alert would fire at the time the problem is occuring. 
The message would then be `operator has finished installing, but has been in a error state for the last 5 minutes` 
wdyt?

Reply:
Agree [changing](https://github.com/integr8ly/integreatly-operator/pull/1519/files#diff-7d10ed51e49fd1fef8a5c0a2f1074fd700809f789237a1dd39cfe136e5655d4fL44) to 5m and removing the rate change on the metrics expression

Reply:
When removing the rate not sure if >=1 makes sense as these metrics `controller_runtime_reconcile_total{controller="installation-controller"}` continuously rise when we don't use rate
![image](https://user-images.githubusercontent.com/16667688/104902203-590d7800-5976-11eb-81d0-c1b4a5b1c5f0.png) making it difficult to trigger these alerts on a specific value

think we should reintroduce rate 5m and check the alert every 10m WDYT @laurafitzgerald ?

Reply:
Also think `>0` might be better to trigger on, having looked at the rates changes above they don't exceed 1 ever.